google/auth/aws.py

```diff
@@ -69,6 +69,8 @@
 _DEFAULT_AWS_REGIONAL_CREDENTIAL_VERIFICATION_URL = (
     "https://sts.{region}.amazonaws.com?Action=GetCallerIdentity&Version=2011-06-15"
 )
+# IMDSV2 session token lifetime. This is set to a low value because the session token is used immediately.
+_IMDSV2_SESSION_TOKEN_TTL = "300"
```

Comment:
Can you include the unit as well?
```suggestion
_IMDSV2_SESSION_TOKEN_TTL _SECONDS= "300"
```